vignettes/adam-adadas.Rmd

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
Why do we need an RMD file for ADaM creation?

Reply:
No, we discussed on Friday to use plain `.R` files.